xbmc/video/VideoDatabase.cpp

```diff
@@ -4696,11 +4802,16 @@ void CVideoDatabase::UpdateTables(int iVersion)
     m_pDS->exec("DROP TABLE IF EXISTS tag");
     m_pDS->exec("ALTER TABLE tagnew RENAME TO tag");
   }
+  if (iVersion < 93)
+  {
+    m_pDS->exec("ALTER TABLE movie ADD c24 integer");
```

Comment:
Please use the constants `VIDEODB_ID_MOVIE_BOOKMARK` and `VIDEODB_ID_MUSICVIDEO_BOOKMARK`.